README.rst

```diff
@@ -35,165 +38,97 @@ You can also create a plot to get a quick overview of the possible performance
-the NAMD configuration file is called ``protein.namd``, you will also need the
-corresponding ``protein.pdb`` and ``protein.psf`` inside the same folder.
-**Warning:** Please be aware that all paths given in the ``protein.namd`` file
-must be absolute paths. This ensures that MDBenchmark does not destroy paths
-when copying files around during benchmark generation.
+Short usage reference
+=====================
 
-In analogy to the GROMACS setup, you can execute the following command to
-generate benchmarks for a module named ``namd/2.12``:
+The following shows a short usage reference for MDBenchmark. Please consult the
+`documentation`_ for a complete guide.
 
-.. code::
-
-    $ mdbenchmark generate --name protein --module namd/2.12 --max-nodes 10
-
-To run benchmarks on GPUs add the ``--gpu`` flag:
-
-.. code::
-
-    $ mdbenchmark generate --name protein --module namd/2.12-gpu --max-nodes 10 --gpu
-
-Be aware that you will need to specify NAMD modules when running GPU and non-GPU
-benchmarks! To work with GPUs, NAMD needs to be compiled separately and will be
-probably named differently on the host of your choice. Using the ``--gpu``
-option on non-GPU builds of NAMD may lead to poorer performance and erroneous
-results.
+Benchmark generation
+--------------------
 
-Usage with multiple modules
----------------------------
+Assuming you want to benchmark GROMACS version 2016.4 and your TPR file is
+called ``protein.tpr``, run the following command::
 
-It is possible to generate benchmarks for different MD engines with a single
-command:
+  mdbenchmark generate --name protein --module gromacs/2016.4
 
-.. code::
+To run benchmarks on GPUs simply add the ``--gpu`` flag::
 
-    $ mdbenchmark generate --name protein --module namd/2.12 --module gromacs/2016.3 --max-nodes 10
+  mdbenchmark generate --name protein --module gromacs/2016.4 --gpu
 
 Benchmark submission
 --------------------
 
-After you generated all benchmarks, you can submit them at once:
-
-.. code::
-
-    $ mdbenchmark submit
+After you generated your benchmarks, you can submit them at once::
 
-To start specific benchmarks separately, use the ``--directory`` option and
-specify the corresponding folder:
-
-.. code::
-
-    $ mdbenchmark submit --directory draco_gromacs/5.1.4-plumed2.3
+  mdbenchmark submit
 
 Benchmark analysis
 ------------------
 
-As soon as the benchmarks have been submitted you can run the analysis script
-via ``mdbenchmark analysis``. When at least one system has finished, the script
-will produce a ``.csv`` output file or a plot for direct usage (via the
-``--plot`` option).
+As soon as the benchmarks have been submitted you can run the analysis via
+``mdbenchmark analysis``. When at least one system has finished, the script will
+produce a ``.csv`` output file or a plot for direct usage (via the ``--plot``
+option).
 
 **Note:** The plotting function currently only allows to plot a CPU and GPU
```

Comment:
Maybe we can point out the fact that a new function i.e. mdbenchmark plot is already mplemented in the develop branch for interested users to test